modules/mirror-registry-troubleshooting.adoc

```diff
@@ -0,0 +1,29 @@
+// module included in the following assembly:
+//
+// * installing-mirroring-creating-registry.adoc
+
+:_content-type: PROCEDURE
+[id="mirror-registry-troubleshooting_{context}"]
+= Troubleshooting mirror registry for Red Hat OpenShift
+
+To assist in troubleshooting _mirror registry for Red Hat OpenShift_, you can gather logs of currently-running systemd services.
+
+.Prerequisites
+
+* You have installed _mirror registry for Red Hat OpenShift_.
+
+.Procedure
+
+* If you installed _mirror registry for Red Hat OpenShift_ with root privileges, you can obtain the list of currently-running system services on the Quay application by entering the following command:
```

Comment:
in terms of systemd, `mirror-registry` installs 4 services:

* quay-app.service
* quay-postgres.service
* quay-redis.service
* quay-pod.service (I don't know what it is, it doesn't look like a long-running service)

`systemctl status quay-app` shows the current status of the quay-app service (even if it has crashed and isn't running)

Ideally it should show logs, but we are not there yet. `podman logs` should be used if you need logs.

Reply:
does "system services" mean "processes/daemons that are running inside the Quay container"?